assets/airports/mkjp.json

```diff
@@ -177,19 +208,184 @@
+            "exitPoints": {
+                "ENEKA": ["ENEKA"]
+            },
+            "draw": [
+                ["GUDIL", "ENEKA*"]
+            ]
+        },
+        "GUDIL5": {
+            "icao": "GUDIL5",
+            "name": "Gudil Five",
+            "rwy": {
+                "MKJP12": ["MLY"],
+                "MKJP30": [["GUDIL", "A28+"]]
+            },
+            "body": [["GUDIL", "A28+"]],
+            "exitPoints": {
+                "LIBEX" : ["LIBEX"],
+                "KEMBO" : ["KEMBO"],
+                "KESPA" : ["KESPA"],
+                "BOSOM": ["BOSOM"]
+            },
+            "draw": [
+                ["GUDIL", "LIBEX*"],
+                ["GUDIL", "KEMBO*"],
+                ["GUDIL", "KESPA*"],
+                ["GUDIL", "BOSOM*"]
+            ]
+        },
+        "KEYNO2": {
+            "icao": "KEYNO2",
+            "name": "Keyno Two",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": [""]
+            },
+            "body": ["KEYNO"],
+            "exitPoints": {
+                "KEYNO": ["KEYNO"]
+            },
+            "draw": [
+                ["KEYNO*"]
+            ]
+        },
+        "LEXUV5": {
+            "icao": "LEXUV5",
+            "name": "Lexuv Five",
+            "rwy": {
+                "MKJP12": [["UKIMO", "A4+"], ["PEMPE", "S220"]],
+                "MKJP30": [""]
+            },
+            "body": ["LEXUV"],
+            "exitPoints": {
+                "KEMBO": ["KEMBO"],
+                "RADOK" : ["RADOK"]
+            },
+            "draw": [
+                ["UKIMO", "PEMPE", "LEXUV*"],
+                ["LEXUV", "KEMBO*"],
+                ["LEXUV", "RADOK*"]
+            ]
+        },
+        "MLY7": {
+            "icao": "MLY7",
+            "name": "Manley Seven",
+            "rwy": {
+                "MKJP12": ["MLY"],
+                "MKJP30": ["MLY"]
+            },
+            "body": ["MLY"],
+            "exitPoints": {
+                "LIBEX": ["LIBEX"],
+                "KEMBO": ["KEMBO"],
+                "KESPA": ["KESPA"],
+                "BOSOM": ["BOSOM"]
+            },
+            "draw": [
+                ["MLY", "LIBEX*"],
+                ["MLY", "KEMBO*"],
+                ["MLY", "KESPA*"],
+                ["MLY", "BOSOM*"]
+            ]
+        },
+        "OMPAL3": {
+            "icao": "OMPAL3",
+            "name": "Ompal Three",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["GUDIl"]
+            },
+            "body": [""],
+            "exitPoints": {
+                "OMPAL": ["OMPAL"]
+            },
+            "draw": [
+                ["GUDIL", "OMPAL*"]
+            ]
+        },
+        "RADOK7": {
+            "icao": "RADOK7",
+            "name": "Radok Seven",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["ELSER"]
+            },
+            "body": ["RADOK"],
```

Comment:
RADOK is defined twice, once in the body and once in the exitPoint. Remove it from either